src/OrchardCore.Build/OrchardCore.Commons.props

```diff
@@ -16,11 +16,29 @@
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <LangVersion>8.0</LangVersion>
 
+    <!-- Add Current Git Commit and Branch to Build  -->
+    <GitCommit Condition="'$(SourceRevisionId)'!=''">$(SourceRevisionId)</GitCommit>
+    <GitBranch Condition="'$(GitBranch)'==''">dev</GitBranch>
+
     <!-- Common NuGet properties-->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <!--This is used to generate symbol package for Source Link-->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     
     <!-- This is used by GitHub Repository to find which repository should contain the package when updloaded -->
     <RepositoryUrl>https://github.com/OrchardCMS/OrchardCore</RepositoryUrl>
+    <PackageProjectUrl>https://www.orchardcore.net/</PackageProjectUrl>
+    <RepositoryCommit>$(GitCommit)</RepositoryCommit>
+    <RepositoryBranch>$(GitBranch)</RepositoryBranch>
     
+    <PackageTags>OrchardCore OrchardCoreCMS CMS MVC AspNetCore</PackageTags>
+    <Copyright>Orchard Core CMS Community and Contributors</Copyright>
```

Comment:
Should there also be a license expression here (BSD 3-Clause) if it's shared will all the packages, at least repo root declares LICENSE with BSD 3-Clause

Reply:
Probably embed the actual license file from repo?

Reply:
NuGet.org supports expressions to show correct link in the UI

Reply:
Done